### PR TITLE
Release/3.1

### PR DESCRIPTION
**Pull request by luca-drf**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -40,8 +38,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'ensembl_prodinf'
-copyright = u'2017 EMBL-EBI'
+project = 'ensembl_prodinf'
+copyright = '2017 EMBL-EBI'
```
Comment (marcoooo) [inline docs/conf.py]: Could you update this to 2020 please ?

File: ensembl/server_utils.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
+import re
+import subprocess
+
+http_uri_regex = r"^(http){1}(s){0,1}(://){1}(.+){1}(:){1}(\d+){1}(/){1}(.+){0,1}$"
```
Comment (marcoooo) [inline ensembl/server_utils.py]: Is it not the file you modified somehow to resolve this uri checks ?

File: ensembl_prodinf/server_utils.py [hunk 3]
```diff
@@ -1,48 +1,54 @@
...
 
 """
 Utilities for dealing with REST and database servers
 """
 
 def assert_http_uri(uri):
     """Check supplied URI matches http"""
-    if not re.search(http_uri_regex, uri):
-        raise ValueError("Endpoint URL doesn't match pattern: http://server_name:port/")
+    parsed_uri = urlparse(uri)
```
Comment (marcoooo) [inline ensembl_prodinf/server_utils.py:16]: This is the one you updated, so the other one is a copy to update as well... waiting for the both files to be merged.

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (luca-drf) [reply docs/conf.py]: See #73

File: ensembl_prodinf/server_utils.py (hunk 3 above)
Comment (luca-drf) [reply ensembl_prodinf/server_utils.py:16]: See #73

**Review by marcoooo — APPROVED** [review]

**Pull request merged**